sdk/healthinsights/Azure.Health.Insights.CancerProfiling/cadl-location.yaml

```diff
@@ -0,0 +1,6 @@
+directory: specification/cognitiveservices/HealthInsights/healthinsights.oncophenotype/
+additionalDirectories:
+  - specification/cognitiveservices/HealthInsights/healthinsights.autorest/
+  - specification/cognitiveservices/HealthInsights/healthinsights.common/
+commit: 7ecc7d7d636976bbe19020a1edd01ce288df734b
+repo: asaflevi-ms/azure-rest-api-specs
```

Comment:
We need to use the spec which already merged into the public repo azure/azure-rest-api-specs. Otherwise, the generated SDK will not be merged into azure-sdk-for-net.

Reply:
Cadl converted to Typespec.